README.md

```diff
@@ -1,2 +1,79 @@
 # Wall-E
 Srishti 2019
+
+![IMG_242133](https://user-images.githubusercontent.com/48998778/55190041-1ee14e00-51c5-11e9-9a4d-d7ac8141bacb.jpg)
+
+
+
+
+## Abstract
+Our main idea was to create a bot which can pick garbage directly from dustbin.
+
+
+## Motivation
+In this present era, people live a very busy life. In such a situation, a person will always look for the way to save
+time.So introducing the robot, WALL-E that can collect waste directly from dustbins and  then dispose the waste to the area which is already predefined. 
+
+
+
+## Mechanical Aspect of the Design
+
+
+Our bot has differential chasis with  three wheels out of which one is castor wheel and the other two motors are connected with two motors which provide movement to the bot.
+With the help of these wheels, the bot can move in all directions.The hands were operated using three dc motors.The "up-down" motion of hands were provided  using worm and spur gear mechanism.The hands were able to converge or diverge with the help of pulley and belt mechanism.
+The body was made using channels and plywood whereas the head was 3D printed.
+
+
+## Electronics Aspect of the Design
+We used two microcontrollers namely Arduino Uno and Arduino Nano, two L298n motor drivers, cytron motor driver,dc motors, servo motor,  bluetooth module and two LED Matrix MAX7219 for the this bot.Uno was used for movements of wheels and hands whereas nano was used for operating the head including LED Matrix used as the eyes of our bot, WallE.
+The arduino code can be found in "src" folder. Our bot was controlled through an Android device using an app made on MITAppInventor2. A bluetooth module was used for transmission of commands from android device to our bot. All the dc motors were powered through Li-Po dc battery.
+
+
+## Cost Structure
+|COMPONENTS | COST(₹)|
+|----|------|
+|Arduino Uno|500|
+|Arduino Nano|300|
+|L298n motor driver(2)|320|
+|Cytron motor driver|2000|
+|Bluetooth module|350|
+|Voltmeter|160|
+|Servo motor(s3003)|400|
+|DC motors(5)|3000|
+|Li-Po battery(2)|2200|
+|Led Matrix MAX7219(2)|300|
+|Other Components|2000|
+
```

Comment:
Add a row that shows the total in the table.

Reply:
Sir, how can I clear commit msgs..?

Reply:
Now its not possible to change the commit messsage. In future take care of that